### PR TITLE
Add SD card flashing instructions

### LINKED ISSUE #257: Add instructions for flashing SD cards

### PR DESCRIPTION
**Pull request by raccube**
Closes #257

### CODE REVIEW

File: kit/brain_board.md [hunk 1]
```diff
@@ -9,3 +9,26 @@ Brain Board
 <img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" />
 The Odroid U3+ Brain Board provided with your kit is what runs the code you write
 and controls the other peripheral boards.
+
+It can be controlled remotely over [WiFi](/docs/kit/wifi).
+
+### Flashing SD card
+
+To fully update your Brain Board's software, or refresh it if you think it's not working correctly, you can flash our SD card image onto the microSD card in your Brain Board.
```
Comment (RealOrangeOne) [inline kit/brain_board.md:17]: We should document how to get to the microSD card

File: kit/brain_board.md [hunk 2]
```diff
@@ -9,3 +9,26 @@ Brain Board
...
+
+To update the SD card, you'll need to download our image, `robot-image.img.xz`. The flashing procedure is identical to flashing Raspberry Pi images.
```
Comment (RealOrangeOne) [inline kit/brain_board.md:19]: Question: Is this definitely the name of the file?

File: kit/brain_board.md [hunk 3]
```diff
@@ -9,3 +9,26 @@ Brain Board
 <img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" />
 The Odroid U3+ Brain Board provided with your kit is what runs the code you write
 and controls the other peripheral boards.
+
+It can be controlled remotely over [WiFi](/docs/kit/wifi).
```
Comment (raccube) [inline kit/brain_board.md:13]: Ignore this: I had an out of date `master` locally.

File: kit/brain_board.md (hunk 2 above)
Comment (raccube) [reply kit/brain_board.md:20]: As we haven't published an image yet, no, but we can change this on release.

File: kit/brain_board.md [hunk 4]
```diff
@@ -9,3 +9,27 @@ Brain Board
 <img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" />
 The Odroid U3+ Brain Board provided with your kit is what runs the code you write
 and controls the other peripheral boards.
+
+### Flashing SD card
+
+In order to get access to the SD card, you will need to open up the case for the brain board. Pull the latch at the end of the case in order to hinge it open.
+The SD card is located on the underside of the board. Grab the SD card with your fingers and simply pull it out of the slot.
```
Comment (RealOrangeOne) [inline kit/brain_board.md:16]: Chore: The brain board is in a case

```suggestion
The SD card is located on the underside of the board. Carefully open the Brain Board's plastic case, grab the SD card with your fingers and simply pull it out of the slot.
```

File: kit/brain_board.md (hunk 2 above)
Comment (trickeydan) [reply kit/brain_board.md:20]: The image names will vary depending on the build number obviously.

We can basically call it whatever we want though.

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**